back/mail-processing/utils/generate_quote.py

```diff
@@ -0,0 +1,111 @@
+from datetime import date
+import random
+
+class GenerateQuote(object):
+    def __init__(self):
+        self.quotes_list = [
+            {
```

Comment:
how about to store quotes in json for easier management?

Reply:
Agreed. Just pushed the changed. I moved the quotes into a file `quotes.json` which is in the static folder.